web/src/components/SharingPopup.tsx

```diff
@@ -170,6 +148,13 @@ const SharingPopupContainer = styled.div`
   position: relative;
 `
 
+const StyledCheckIcon = withIconStyle(CheckIcon)
+const StyledCopyIcon = withIconStyle(ContentCopyIcon)
+const StyledWhatsAppIcon = withIconStyle(WhatsAppIcon)
+const StyledMailIcon = withIconStyle(MailOutlinedIcon)
+const StyledFacebookIcon = withIconStyle(FacebookOutlinedIcon)
+const StyledCloseIcon = withIconStyle(CloseIcon)
+
```

Comment:
:thought_balloon: how about we put `IconButton` into it's own file with the Icon as a child like the native one ?
and just pass the Icon for each.

Reply:
Can't you apply these styles at the icon component? (in side the component file) why would it need a `sx` for each?